README.md

```diff
@@ -156,6 +157,13 @@ links) will be posted to the thread.
   (`restricted_action` error). Give your app explicit permission to post using
   the channel settings if necessary.
 
+### Datastores
+This app uses two datastores: for `announcements` and for `drafts`.
+
+Definition files for these can be found in `datastores/`.
+To use a datastore, an app needs to have `datastore:write`/`datastore:read` scopes,
+which you can see are present in this app's `manifest.ts` file. 
```

Comment:
```suggestion
which are present in this app's `manifest.ts` file. 
```

Reply:
Looks like also the Deno formatting check has failed so I think that needs to be updated before merging as well!

Reply:
Thank you! I've pushed up the changes and confirmed we've passed the formatting check. 🙇‍♀️